conjure-test/src/test/servers.rs

```diff
@@ -949,6 +958,17 @@ fn custom_path_params() {
         .send_sync("path_params");
 }
 
+#[test]
+fn wildcard_path_param() {
+    let mut mock = MockCustomService::new();
+    mock.expect_wildcard_path_param()
+        .with(eq("some/wildcard/match".to_string()))
+        .returning(|_| Ok(()));
+    Call::new(CustomServiceEndpoints::new(mock))
+        .path_param("path", "some/wildcard/match")
+        .send_sync("wildcard_path_param");
```

Comment:
Can you also have this look at the PathSegments and confirm that the regex pattern is set there?

Reply:
Added that, also ended up realizing that you have a regex in one of the client macros in a test. I know we chatted about not allowing those for clients, curious if you have further thoughts on that?

Reply:
Sorry, not for clients, it's here where we have a ".+" regex pattern: https://github.com/palantir/conjure-rust/blob/b900cffd553a1a9f6a82eacd109e9a21af849a78/conjure-codegen/src/example_types/another/test_service.rs#L1372